src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1655,6 +1658,7 @@ class ContentExplorer extends Component<Props, State> {
                             parentElement={this.rootElement}
                             appElement={this.appElement}
                             onUpload={onUpload}
+                            contentUploaderProps={contentUploaderProps}
```

Comment:
This seems a little odd, since a lot of the sibling props here are constituents of `contentUploaderProps`. What are the actual props we need to pass in to support enabling/disabling of chunked uploads per the customer's needs?

Reply:
This seems to follow the approach with `contentPreviewProps` and the hierarchy of props following that convention. For this specific customer case, we only need to expose `chunked` option, but other ContentUploader props (e.g. `fileLimit`) make sense to be options of the Content Explorer, and I think they should be nested contentUploaderProps and not top-level Content Explorer options.

Reply:
I see the argument to expose it as its own thing; my concern was passing them into UploadDialog rather than spreading them, but it looks like PreviewDialog works the same way.